_CodingChallenges/134-heart-curve.md

```diff
@@ -17,6 +17,19 @@ videos:
     url: "/Tutorials/9-additional-topics/9.20-polar-coordinates"
   - title: "Drawing with Fourier Transform and Epicycles"
     url: "/CodingChallenges/130.1-fourier-transform-drawing"
+    
+    
+contributions:
+  - title: "Changing Cassini Oval to Heart"
+    author: Simon Oliehoek
+    name: "Simon"
+    url: ""
+    url: "https://i.imgur.com/mXMyw9X.gifv"
+    source: "https://github.com/Serpelic/Processing_Projects/tree/master/Changing_heart"
+    
 ---
 
 Using a parametric equation from Wolfram Mathworld (and polar to cartesian coordinate transformation), I draw a heart curve in Processing (Java).
+
+
```

Comment:
only one line

Reply:
How where what. Just a sec, I've never used github before.

Reply:
Done?

Reply:
remove the extra line, you only need one line here. (yeah it also happened to me at first contrib)